alcap/configure/geometry_R2015b/left-package_P2

```diff
@@ -19,7 +19,9 @@ DEFINE: thDLSi 0.0005 # thickness of the Si dead layer
 DEFINE: thDLAl 0.0005 # thickness of the Al dead layer
 
 DEFINE: nomThSiL1 0.052 # nominal SiL1 thickness (dead layer needs to be included in this)
-DEFINE: thSiL1  nomThSiL1 # -2*thDLSi # active SiL1 thickness
+#DEFINE: nomThSiL1 0.054
+DEFINE: thDLSiL1 thDLSi
+DEFINE: thSiL1  nomThSiL1-thDLSi-thDLSiL1-2*thDLAl # active SiL1 thickness
```

Comment:
I didn't do the math but what the latest value of thSiL1?

Reply:
Nominal thickness should still be 52 um. Actual thickness will be 50 um (0.5 um dead layer of Si and Al on front and back)